### PR TITLE
Recursive outlines

### PR DESCRIPTION
**Pull request by chaselambda**
OPML allows for nested outlines, so I've implemented them.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "opml-generator",
-  "version": "1.0.2",
+  "name": "opml-generator2",
```
Comment (azu) [inline package.json]: ??

please revert these commits

https://github.com/theicfire/opml-generator/commit/85c9b412fbad741e01af2b328e1acf1368d818c5
https://github.com/theicfire/opml-generator/commit/769caeaa5b3b47853eb8d9c832e3dd2027d371c8
https://github.com/theicfire/opml-generator/commit/0b8e729c50303cdc6d48a98afc89d0349e270f48

OR

```
git reset --hard dc71a3b925db7ba7e849d552aa18202c1ca84147
git push -f theicfire recursive
```

**Comment by azu** [comment]
looks good without above the comment.

**Comment by chaselambda** [comment]
Oh sorry about those other commits! I was making a new package for myself to use while waiting for this one. I didn't realize this was going up to the pull request. Fixed.

**Comment by azu** [comment]
Release opml-generator@1.1.0

@theicfire Thanks for great work! :+1:

**Pull request merged**